config/application.yml

```diff
@@ -0,0 +1,11 @@
+default: &default
```

Comment:
I added it but it didn't work, going to look into fixing that. However, it doesn't technically need to be in there because it's not sharing my actual token and only displays the default test token.